docs/develop/01_sdk/04_integrate/04_extension_api.md

```diff
@@ -151,43 +139,69 @@ The following is an example of how you can use this method in a React applicatio
   </TabItem>
 </Tabs>
 
-## Well-Known DID Configuration
+## DID Configuration
+
+You need [an existing DID configuration](../../01_sdk/02_cookbook/01_dids/00_generate_keys.md) setup for your application to communicate with KILT extensions.
```

Comment:
I'd say that you need an active DID with which it must sign a Did Configuration Resource to establish the association of your DID and your application's domain.

Reply:
linking to how you create a DID makes a lot of sense though.